webui/react/src/components/JupyterLab.settings.ts

```diff
@@ -0,0 +1,60 @@
+import { number, string, undefined as undefinedType, union } from 'io-ts';
+
+import { ShortcutConfig } from 'hooks/useKeyTracker';
+import { SettingsConfig } from 'hooks/useSettings';
+import { JupyterLabOptions } from 'utils/jupyter';
+
+const STORAGE_PATH = 'jupyter-lab';
+export const DEFAULT_SLOT_COUNT = 1;
+
+const defaultShortcut: ShortcutConfig = {
+  altKey: false,
+  ctrlKey: false,
+  key: 'g',
```

Comment:
Note that "Super + G" is a shortcut for searching in Firefox and Chrome ("Go to line"). May want to go with a different default.

Reply:
general question: what if user's keybinding shortcut conflicts with the browser shortcut, do we wanna trigger both or just web?

Reply:
Super + Shift + g should be fine though, right?

Reply:
it seems to trigger even with shift. I'll go ahead and change it to Super+Shift+L